index.html

```diff
@@ -97,20 +97,34 @@ <h2>Stages</h2>
     <td>Experimental
   </tr>
   <tr>
-    <td>3
-    <td>Indicate that further refinement will require feedback from implementations and users
+    <td>2.7
+    <td>Gain initial experience with the feature through testing and prototyping
     <td>
       <ul>
         <li>Above
         <li>Complete spec text
         <li>Designated reviewers have signed off on the current spec text
-        <li>All ECMAScript editors have signed off on the current spec text
+        <li>ECMAScript editors have signed off on the current spec text
+      </ul>
+    </td>
+    <td>The solution is complete and no further work is possible without feedback from tests, implementations, or usage. No changes to the proposal will be requested aside from those elicited through testing or implementation experience.
+    <td>Complete: all semantics, syntax and API are completely described
+    <td>Limited: only those resulting from new information obtained through testing or usage experience
+    <td>Spec compliant prototypes
+  </tr>
+  <tr>
+    <td>3
+    <td>Indicate that further refinement will require feedback from implementations or users
+    <td>
+      <ul>
+        <li>Above
+        <li>The feature has sufficient testing and appropriate pre-implementation experience
       </ul>
     </td>
     <td>The solution is complete and no further work is possible without implementation experience, significant usage and external feedback.
     <td>Complete: all semantics, syntax and API are completely described
     <td>Limited: only those deemed critical based on implementation experience
-    <td>Spec compliant
+    <td>Spec compliant production
```

Comment:
```suggestion
    <td>Spec compliant; safe to use in production
```
maybe?

Reply:
Since we got consensus on specifically this wording, I'm going to leave it alone. I'll be submitting a follow-up PR soon with names and a refactoring of descriptions that maintains the spirit here, which will include addressing this wording.